### DISCUSSION TITLE
Act 2 The Vaal Ruins Shape

### DISCUSSION THREAD

**Discussion by GamerCH** [Lailloken/Exile-UI#418] [Ideas & Feedback]
Found another Shape (o shape) in Act 2 The Vaal Ruins. Is this rare or just new Implemented from GGG?
![leilloken o shape](https://github.com/user-attachments/assets/e4fd97fc-45a8-4a98-9eb5-b945213f6fcc)

**Comment by Lailloken** [comment]
Hmm, not sure. I don't think I've seen that before. The guide is written by HeartofPhos, and I think it's mostly based on popular racing strats and crowd-sourced resources.

**Comment by GamerCH** [comment]
Ok i opend a Discussion over there https://github.com/HeartofPhos/exile-leveling/issues/101
